charts/enterprise-kyverno-operator/README.md

```diff
@@ -93,4 +92,7 @@ helm uninstall -n enterprise-kyverno-operator enterprise-kyverno-operator
 | kyverno.image.pullPolicy | string | `"IfNotPresent"` | Kyverno Image pull policy |
 | kyverno.image.tag | string | `v1.9.1-n4k.nirmata.1` | Image tag (defaults to chart app version) |
 | kyverno.helm | object | `helm.rbac.serviceAccount.name=kyverno` | Free form yaml section with helm parameters in Kyverno chart. See all parameters [here](https://github.com/nirmata/kyverno-charts/tree/main/charts/nirmata#values) |
-| policies.policySets | list | `[{name: best-practices, type: helm, chartRepo: https://nirmata.github.io/kyverno-charts, chartName: best-practice-policies, version: 0.1.0}, {name: pod-security, type: helm, chartRepo: https://nirmata.github.io/kyverno-charts, chartName: pod-security-policies, version: 0.1.0}]` | Default policy sets to install along with operator |
+| policies.enableBestPractices | bool | `true` | Whether default best practices policy set is to be installed along with operator |
```

Comment:
Should this be kyverno.policies.enableBestPractices?

Reply:
Actually, the policies section is no longer under kyverno. Plus Anusha suggested a further change to spec format. Added that now.